src/views/Prescriptions/Entity/PatientCard/index.tsx

```diff
@@ -11,15 +11,19 @@ interface OwnProps {
   loading: boolean;
 }
 
-const PatientCard = ({ prescription, loading }: OwnProps) => (
-  <Card title={intl.get('screen.prescription.entity.patient.card.title')} data-cy="PatientCard">
-    <ParagraphLoader loading={loading} paragraph={{ rows: 6 }}>
-      <PatientContent
-        patient={prescription?.subject.resource!}
-        reference={prescription?.requester?.organization?.reference}
-      />
-    </ParagraphLoader>
-  </Card>
-);
+const PatientCard = ({ prescription, loading }: OwnProps) => {
+  const patient = prescription?.subject.resource!;
+  return (
+    <Card title={intl.get('screen.prescription.entity.patient.card.title')} data-cy="PatientCard">
+      <ParagraphLoader loading={loading} paragraph={{ rows: 6 }}>
+        <PatientContent
+          patient={patient}
+          isPrenatal={prescription?.category && prescription.category[0]?.text === 'Prenatal'}
```

Comment:
Désolée c'est lundi matin... pourquoi ne pas écrire juste : 
```suggestion
          isPrenatal={prescription?.category?.[0]?.text === 'Prenatal'}
```

Reply:
ah, en effet, meilleure idée. merci